velox/functions/sparksql/fuzzer/CMakeLists.txt

```diff
@@ -12,15 +12,83 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Set up Spark connect
+file(
+  GLOB PROTO_FILES
+  RELATIVE ${PROJECT_SOURCE_DIR}
+  CONFIGURE_DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/proto/spark/connect/*.proto)
+foreach(PROTO ${PROTO_FILES})
```

Comment:
Indeed they are similar but it seems only the 'add_custom_command' can be reused. How do you think? Thanks.

1. They define different variables for the paths of outputs. Spark query runner defines 'GRPC_SRCS' and 'GRPC_HDRS' in addition.
2. The proto files are downloaded from Spark at compile time, so their include path does not contain velox relative paths like [plan.proto](https://github.com/facebookincubator/velox/blob/main/velox/substrait/proto/substrait/plan.proto#L6). This causes difference on the 'PROTO_PATH'.

The detailed difference is as below. Left is dwio and right is spark query runner.
![image](https://github.com/user-attachments/assets/16419dfc-ae86-4ed8-8b8e-3e1954008ae9)

Reply:
I think the official `Findprotobuf.cmake` comes with a utility for this but we are not using it because of `resolve_dependency` (I assume, I'd have to look into it). There is likely a way to still make that work, I can check that out if you'd like @kgpai ?

Reply:
@assignUser That would be great if you have the bandwidth .

Reply:
@assignUser FYI, https://cmake.org/cmake/help/latest/module/FindProtobuf.html has examples to generate both proto and grpc files.